### PR TITLE
AS-4155: skip records with invalid country codes

### PR DESCRIPTION
**Pull request by rahul-r-jain**

### CODE REVIEW

File: nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java [hunk 1]
```diff
@@ -9,7 +9,7 @@ public class CountryCodeMapper {
 
     public static String getCountryCode(String code) {
         CountryCode countryCode = CountryCode.getByCode(code);
```
Comment (iagoquirino) [inline nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java:11]: will this not throw exception?

Comment (rahul-r-jain) [reply nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java:11]: nope.. it just returns `null` ([code](https://github.com/nutmegdevelopment/nm-salesforce-nationalities-service/blob/ebfef9842e1661dc8ce63c7f02874da46d9dd7f7/nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/dto/CountryCode.java#L815))

Comment (rahul-r-jain) [reply nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java:11]: if needed I can throw exception in `CountryCode` and add try catch in `CountryCodeMapper`

**Review by iagoquirino — APPROVED** [review]

File: nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java (hunk 1 above)
Comment (iagoquirino) [reply nm-salesforce-nationalities-topology/src/main/java/com/nutmeg/salesforce/nationalities/stream/mapper/CountryCodeMapper.java:11]: nah no need to worry, I thought just to double check

**Pull request merged**